### PR TITLE
feat(logging): Add the new custom process output class and inject it into the default list renderer options

### PR DESCRIPTION
**Pull request by instamenta**
## Description

* Add the new custom process output class and inject it into the default list renderer options
* Fix warnings in IDE - ( type mismatches, etc. ) 
* Add delay on the Node.checkNetworkNodeActiveness since it prevents the gRPC service error

### Related Issues

* Closes #
  * [581](https://github.com/hashgraph/solo/issues/581)
  * [546](https://github.com/hashgraph/solo/issues/546)

### CODE REVIEW

File: src/core/process_output.mjs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  write (chunk, encoding, fd) {
+    const message = chunk.toString()
+
+    // Capture stdout as debug, stderr as error
+    if (fd === process.stdout.fd) {
+      this._logger.debug(`Listr Process stdout: ${message}`)
+    } else if (fd === process.stderr.fd) {
+      this._logger.error(`Listr Process stderr: ${message}`)
+    } else {
+      this._logger.info(`Listr Process log: ${message}`)
```
Comment (jeromy-cannon) [inline src/core/process_output.mjs:42]: When I run `mirror_node.test.mjs` and do a search in ~/.solo/logs/solo.log, I can't find any references to `Listr Process`

Comment (instamenta) [reply src/core/process_output.mjs]: I extended the method's base method which seems to fix the issue

File: src/core/process_output.mjs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+export class CustomProcessOutput extends ProcessOutput {
+  /** @param {SoloLogger} logger */
+  constructor (logger) {
+    super()
+    /** @private */
+    this._logger = logger
+  }
+
+  /** @inheritDoc */
+  toStdout (chunk, eol = true) {
```
Comment (jeromy-cannon) [inline src/core/process_output.mjs:34]: can we do a split on '\n' to turn it into multiple log statements? 
from this:
```
{"label":"","level":"DEBUG","message":"\u001b[32m✔\u001b[39m Initialize\n\u001b[32m✔\u001b[39m Prepare chart values\n\u001b[32m✔\u001b[39m Install 'fullstack-cluster-setup' chart \u001b[32m[17s]\u001b[39m","ms":"+2ms","timestamp":"2024-09-30T14:21:44.573Z","traceId":"84878b72-68f2-4710-911d-94e02cc9a1bc"}
```
which I see using JQ as this:
```
2024-09-30T14:21:44.573Z	✔ Initialize\n✔ Prepare chart values\n✔ Install 'fullstack-cluster-setup' chart [17s]
```
into this:
```
2024-09-30T14:21:44.573Z	✔ Initialize
2024-09-30T14:21:44.573Z	✔ Prepare chart values
2024-09-30T14:21:44.573Z	✔ Install 'fullstack-cluster-setup' chart [17s]
```

Comment (JeffreyDallas) [reply src/core/process_output.mjs:34]: Fixed

**Review by jeromy-cannon — APPROVED** [review]

**Pull request merged**